### PR TITLE
Initial changes

### PR DESCRIPTION
**Pull request by danial-rokt**
Initial changes

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# references:
+# * https://www.objc.io/issues/6-build-tools/travis-ci/
```
Comment (djseo-rokt) [inline .travis.yml:2]: It's hard for me to believe we use travis.

event if we use it, osx_image is too old and the language would not be matched

**Comment by danial-rokt** [comment]
> FBSnapshotTestCase
At the moment we don't use snapshot test but I was thinking if we want to have a UI test, SnapshotTest would be a good choice. Will remove it in future If I don't use it.

File: RoktWebViewSDK.podspec [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+#   * Finally, don't worry about the indent, CocoaPods strips it!
+
+  s.description      = <<-DESC
+TODO: Add long description of the pod here.
+                       DESC
+
+  s.homepage         = 'https://github.com/ROKT/rokt-webview-sdk-ios.git'
+  # s.screenshots     = 'www.example.com/screenshots_1', 'www.example.com/screenshots_2'
+  s.license          = { :type => 'Copyright 2020 Rokt Pte Ltd', :file => 'LICENSE.md' }
+  s.author           = { 'Danial Motahari' => 'danial.motahari@rokt.com' }
```
Comment (djseo-rokt) [inline RoktWebViewSDK.podspec:27]: Shouldn't this be nativeapps@rokt.com?

File: RoktWebViewSDK.podspec [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  s.source           = { :git => 'https://github.com/Danial Motahari/RoktWebViewSDK.git', :tag => s.version.to_s }
```
Comment (djseo-rokt) [inline RoktWebViewSDK.podspec:28]: This one should be the current repo?

File: RoktWebViewSDK.podspec [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  # s.social_media_url = 'https://twitter.com/<TWITTER_USERNAME>'
+
+  s.ios.deployment_target = '8.0'
```
Comment (djseo-rokt) [inline RoktWebViewSDK.podspec]: The lower the better! I'm happy with 8. =)

**Pull request merged**